### PR TITLE
Transcriptome assembly lesson (Trinity)

### PR DESCRIPTION
**Pull request by fpsom**
Still missing learning objectives
The longest step is ~20min

- [x] tutorial resets working directory at beginning with a `cd ~/``
- [x] tutorial contains relevant `conda install -y` commands at the beginning
- [ ] tutorial links to previous tutorial (at top) and next tutorial (at bottom)
- [ ] tutorial has learning objectives at top

### CODE REVIEW

File: transcriptome-assembly.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Transcriptome assembly - some basics
+
+Learning objectives:
```
Comment (adamjorr) [inline transcriptome-assembly.md:3]: are these accurate? your comment implied they were missing

File: transcriptome-assembly.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	* Different types of assemblies
+	* How do assemblers work?
+	* Checking the quality of assembly
+	* Understanding Transcriptome assembly
+	
+In [variant calling](http://angus.readthedocs.io/en/2018/mapping-variant-calling.html), we mapped reads to a reference and looked systematically for differences.
+
+But what if you don't have a reference? How do you construct one?
+
+The answer is *de novo* assembly, and the basic idea is you feed in your reads and you get out a bunch of *contigs*, that represent stretches of DNA present in the reads that don't have any long repeats or much significant polymorphism.  Like everything else, the basic idea is that you run a program, feed in the reads, and get out a pile of assembled DNA.
```
Comment (adamjorr) [inline transcriptome-assembly.md]: DNA -> RNA

File: transcriptome-assembly.md (hunk 1 above)
Comment (fpsom) [reply transcriptome-assembly.md:3]: Yes it is accurate. I simply copied the learning objectives from the genome assembly, so they should be re-assessed.

File: transcriptome-assembly.md (hunk 2 above)
Comment (fpsom) [reply transcriptome-assembly.md]: fixed.

File: transcriptome-assembly.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+cp yeast_trinity/Trinity.fasta yeast-transcriptome-assembly.fa
+``` 
+ 
+Now, look at the beginning:
+
+```
+head yeast-transcriptome-assembly.fa
+```
+    
+It's DNA! Yay!
```
Comment (adamjorr) [inline transcriptome-assembly.md]: RNA ?

Comment (fpsom) [reply transcriptome-assembly.md]: Fixed. :)

**Review by adamjorr — COMMENTED** [review]
Looks good. Took 23 min.

**Review by adamjorr — APPROVED** [review]

**Pull request merged**